app/Http/Controllers/Airline/GetAirlineController.php

```diff
@@ -3,12 +3,13 @@
 namespace App\Http\Controllers\Airline;
 
 use App\Http\ViewModels\AirlineViewModel;
+use App\Transformers\AirlineTransformer;
 use Illuminate\Http\JsonResponse;
 
 class GetAirlineController
 {
     public function __invoke(AirlineViewModel $viewModel): JsonResponse
```

Comment:
This should recieve an Airline model really, and just return it using the transformer.
The view model would just be used in the controllers returning a view, like the index in your case